### ISSUE TITLE
Documentation Fix Regarding Beta-Node

### ISSUE THREAD

**Issue by georgimld** [openkfw/TruBudget#1701]
- [x] I have searched the [issues](https://github.com/openkfw/TruBudget/issues) of this repository and believe that this is not a duplicate.

## Description 😯
The documentation regarding conneciton a beta node to an alpha node seems to be contradicting in the following links
- https://trubudget.net/docs/operation-administration/installation/connect-to-an-existing-network/docker#:~:text=EXTERNAL_IP,the%20alpha%20blockchain 
and 
- https://github.com/openkfw/TruBudget/blob/main/blockchain/environment-variables.md#:~:text=The%20IP%20address%20with%20which%20the%20current%20node%20can%20be%20reached

Basically it should be "you set the EXTERNAL_IP to 53.53.53.53" [here](https://github.com/openkfw/TruBudget/blob/main/blockchain/environment-variables.md#:~:text=the%20EXTERNAL_IP%20to-,52.52.52.52,-on%20this%20node)

But check it to be sure
